.github/actions/generate_manifest/schema.py

```diff
@@ -14,18 +14,23 @@
 """Schema definition for manifest.json."""
 
 from datetime import datetime
-from typing import List, Literal
+from typing import List, Literal, Optional
 
 from pydantic import BaseModel
 
 
+class PackageExtraMetadata(BaseModel):
+    patch_base_version: str
```

Comment:
Ah for extra metadata, all fields should be optional, let's change it to `patch_base_version: Optional[str] = None`

Reply:
Good catch, fixed.